src/components/form/SwitchField.tsx

```diff
@@ -32,7 +32,11 @@ export const SwitchField: FC<SwitchFieldProps> = ({
   ...props
 }) => {
   const name = useFieldName(nameProp);
-  const { input, meta, rest } = useField(name, { defaultValue, ...props });
+  const { input, meta, rest } = useField(name, {
+    defaultValue,
+    type: 'checkbox',
+    ...props,
```

Comment:
We should probably enforce this type right?
```suggestion
    ...props,
    type: 'checkbox',
```

Reply:
Yar.